### DISCUSSION TITLE
Thanks Guys!

### DISCUSSION THREAD

**Discussion by cube-dan** [strapi/strapi#6229] [Legacy Topics]
I just wanted to say thanks for strapi and especially for the docs and guides; like: https://strapi.io/documentation/3.0.0-beta.x/guides/api-token.html 

Using any framework/cms ...thing is a daunting task when bringing yourself up to speed but you guys have made it a pleasure to work with both the code (how it's structured) and how we'll it's all documented. 

Keep up the great work!

**Comment by lauriejim** [comment]
Thank you for this feedback!
We are doing our best to offer the best project possible.
I'm sharing this with the team.
